content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -22,61 +23,62 @@ This document describes the current state of `VolumeSnapshots` in Kubernetes. Fa
 
 Similar to how API resources `PersistentVolume` and `PersistentVolumeClaim` are used to provision volumes for users and administrators, `VolumeSnapshotContent` and `VolumeSnapshot` API resources are provided to create volume snapshots for users and administrators.
 
-A `VolumeSnapshotContent` is a snapshot taken from a volume in the cluster that has been provisioned by an administrator. It is a resource in the cluster just like a PersistentVolume is a cluster resource.
+A `VolumeSnapshotContent` is a snapshot of a volume in the cluster that has been provisioned by an administrator. It is a resource in the cluster just like a `PersistentVolume` is a cluster resource.
 
-A `VolumeSnapshot` is a request for snapshot of a volume by a user. It is similar to a PersistentVolumeClaim.
+A `VolumeSnapshot` is a request for snapshot of a volume by a user. It is similar to a `PersistentVolumeClaim`.
 
-While `VolumeSnapshots` allow a user to consume abstract storage resources, cluster administrators
-need to be able to offer a variety of `VolumeSnapshotContents` without exposing
-users to the details of how those volume snapshots should be provisioned. For these needs
+Cluster administrators need to be able to offer different ways in
+which volume snapshots can be created without exposing
+users to the low level details. For these needs
 there is the `VolumeSnapshotClass` resource.
 
 Users need to be aware of the following when using this feature:
 
-* API Objects `VolumeSnapshot`, `VolumeSnapshotContent`, and `VolumeSnapshotClass` are CRDs, not part of the core API.
+* API Objects `VolumeSnapshot`, `VolumeSnapshotContent`, and `VolumeSnapshotClass` are CRDs, not part of the core API. The CSI drivers that support the volume snapshot functionality will automatically install these required CRDs.
+
```

Comment:
FYI, CSI drivers won't automatically install the snapshot CRDs after snapshot APIs become Beta.  So this sentence will be removed after the snapshot API going to Beta work is complete (planned for 1.17).

Reply:
Add a note that the CRD install will be decoupled in the future

Reply:
Done.